library/network_connections.py

```diff
@@ -2193,6 +2197,20 @@ def run_action_up(self, idx):
 ###############################################################################
 
 
+def get_client_version_tuple(client_version):
```

Comment:
Suggested new name:

`version_to_tuple(version)`

This is not specific for the client version but works for any dot-separated version string.